insights/parsers/bond.py

```diff
@@ -38,22 +38,22 @@
 ``/proc/net/bonding``.
 
 Examples:
-    >>> bond_info = shared[Bond]
+    >>> type(bond_info)
+    <class 'insights.parsers.bond.Bond'>
     >>> bond_info.bond_mode
-    '0'
+    '4'
     >>> bond_info.partner_mac_address
-    None
+    '00:00:00:00:00:00'
     >>> bond_info.slave_interface
-    ['eno1', 'eno2']
+    ['eth1', 'eth2']
```

Comment:
Same to above

Reply:
@xiangce - Input data is different, that's why the changes are appeared.